decidim/decidim
# Add multiple statuses on proposals

#### :tophat: What? Why?
*Please describe your pull request.*

#### :pushpin: Related Issues
*Link your PR to an issue*
- Related to [DDP31]
- Meta Proposal: [Expand participatory process status types](https://meta.decidim.org/processes/roadmap/f/122/proposals/15920)

#### Testing
1. Reacreate  your DB 
2. Visit proposals's index page in admin panel 
3. See there is a new button named "States"
4. Edit and Add various states 
5. Go to Proposals answer page and add and answer, including new States. 
6. Publish the answer, and see is available in interface
7. Also, you can use Proposal templates 

### :camera: Screenshots
*Please add screenshots of the changes you are proposing*

![image](https://github.com/tremend-cofe/decidim/assets/105683/4ec2e344-6a03-456f-b33f-aca1155a00d4)
![image](https://github.com/tremend-cofe/decidim/assets/105683/010751ea-8d04-4365-ab0f-5a2b2a6a80a5)
![image](https://github.com/tremend-cofe/decidim/assets/105683/66e75fc6-5606-4bd1-aee8-04f37d1c3e8f)
![image](https://github.com/tremend-cofe/decidim/assets/105683/bf1725fc-2638-4fdd-a1b8-77c6570f3684)
![image](https://github.com/decidim/decidim/assets/105683/7140473b-8e6f-41e7-8a68-ebe3e8de0de2)


:hearts: Thank you!
